lnp2p/msg.go

```diff
@@ -57,7 +57,7 @@ func sendMessages(queue chan outgoingmsg) {
 		// Make sure the connection isn't closed.  This can happen if the message was queued but then we disconnected from the peer before it was sent.
 		conn := recv.peer.conn
 		if conn == nil {
-			logging.Warnf("peermgr: Tried to send message to disconnected peer %s\n", recv.peer.GetPrettyName())
+			logging.Warnf("peermgr: Tried to send message to disconnected peer %d, %s\n", recv.peer.Idx, recv.peer.Nickname)
```

Comment:
If `Nickname` isn't defined then it doesn't show anything, `GetPrettyName` returns a truncated version of the addr if there's no nickname defined.

Reply:
Ah, this is another debate in itself. The reason why I changed it is because of users themselves. Say users give the same pretty name to both, you don't want one guy masquerading as another, etc. But again, not a "strict" change by itself, so can add it back in.

Reply:
Well if nicknames are only local labels it shouldn't matter?  It's your own fault if you give the same nickname to two nodes.

Reply:
Agreed, the idea is that there must atleast be a log fallback (hence the change) so that they understand it later (and makes it easier for them to debug). I personally don't like nicknames for nodes, so this view could be a result of that as well, so I'm open to revert if others feel appropriate.

Reply:
Perhaps we should have a `ResolvePeer(string) -> *Peer` function on `PeerManager` that takes looks up a literal address or falls back to searching nicknames if that fails.